.yarnrc.yml

```diff
@@ -1,13 +1,17 @@
 enableGlobalCache: true
 
+httpProxy: "http://sia-lb.telekom.de:8080"
```

Comment:
Not sure this file should be changed 🤔

Reply:
I removed this already from the changes. GitHub shows the file "outdated" here in your review 🤔 and it's also not visible in the list of changes of the PR, anymore.